apps/rush-lib/src/cli/actions/InstallAction.ts

```diff
@@ -34,7 +35,10 @@ export class InstallAction extends BaseInstallAction {
       recheckShrinkwrap: false,
       networkConcurrency: this._networkConcurrencyParameter.value,
       collectLogFile: this._debugPackageManagerParameter.value!,
-      variant: this._variant.value
+      variant: this._variant.value,
+      maxInstallAttempts: this._maxInstallAttempts.value !== undefined
```

Comment:
If we change `_maxInstallAttempts` to use `defaultValue`, I think we can remove these checks.